lib/passpartu/check_policy.rb

```diff
@@ -0,0 +1,59 @@
+module Passpartu
+  class CheckPolicy
```

Comment:
I meant, move waterfall logic into separate module, but not the whole verification logic. `passaprtu/verify` it's a core, and `passpartu/waterfall_rules` should be a component!